src/dyad/common/dyad_structures.h

```diff
@@ -20,6 +20,7 @@ struct dyad_ctx {
     flux_t* h;                      // the Flux handle for DYAD
     struct dyad_dtl* dtl_handle;    // Opaque handle to DTL info
     const char* fname;              // Used to track which file is getting processed.
+    bool use_fs_locks;              // Used to track if fs locks should be used.
```

Comment:
Make sure the compatibility with the python binding as well as the initialization of the struct in dyad_core.c

Reply:
1. https://github.com/flux-framework/dyad/blob/6d3df1b5f680ca7295a1e46e7aa731a9982636e6/pydyad/pydyad/bindings.py#L24
2. https://github.com/flux-framework/dyad/blob/9a87cf0ac93b982fd0eac5857c0b175c9ce70f68/src/dyad/core/dyad_core.c#L38

Reply:
Added